thirdparty/rabbitmq-server/zimbra-rabbitmq-server/debian/control

```diff
@@ -0,0 +1,14 @@
+Source: zimbra-rabbitmq-server
+Build-Depends: debhelper (>= 9), m4, dpkg-dev (>= 1.15.7), zimbra-erlang (>= 25.0.3-1zimbra8.8b1ZAPPEND), zimbra-openssl-dev (>= 1.1.1q-1zimbra8.7b4ZAPPEND), zip, rsync, gzip, sed
```

Comment:
should we add `zip, rsync, gzip, sed` packages to `pkg-add` target in makefile to install before compiling this?

Reply:
or we can create RHEL.def and UBUNTU.def files if OS wise these dependencies will be different

Reply:
yes , need to add in pkg-add. Committed required changes